### ISSUE TITLE
【提问】Merkle root是否就是区块的哈希值？

### ISSUE THREAD

**Issue by NTCNDL** [FISCO-BCOS/FISCO-BCOS#4134]

**Comment by LucasLi1024** [comment]
不是。区块头的字段txsRoot记录区块内所有交易MerkleRoot的哈希。区块数据结构文档https://fisco-bcos-doc.readthedocs.io/zh-cn/latest/docs/design/protocol_description.html?highlight=Merkleroot#blockheaderdata
